modules/op-creating-mounting-kms-authentication-token-secret.adoc

```diff
@@ -0,0 +1,56 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-mounting-kms-authentication-token-secret_{context}"]
+= Creating and mounting the KMS authentication token secret
+
+You can provide the authentication token for the KMS server (the value of `VAULT_TOKEN`) using a secret. You must create this secret, mount it on the {tekton-chains} controller, and set the `signers.kms.auth.token-dir` parameter to the directory where the secret is mounted.
```

Comment:
kindly consider updating to "You can provide the authentication token for the KMS provider (for example the value of VAULT_TOKEN if the KMS provider is Hashicorp Vault) using a secret."